### ISSUE TITLE
Screen Blank Cant select patch event to see more details with 1.13.0

### ISSUE THREAD

**Issue by rverma-nikiai** [spinnaker/spinnaker#4204]
### Issue Summary:
Cant select patch event to see more details with 1.13.0

### Cloud Provider(s):
AWS

### Environment:
Kuberentes 1.11.5

### Feature Area (if this issue is UI/UX related, please tag `@spinnaker/ui-ux-team`):
Bug

### Description:
Cant select patch event to see more details with 1.13.0

### Steps to Reproduce:
Create a pipeline with step patch-env with any kind but parameters variable. Click on patch step and see the screen go blank.

### Additional Details:
Very high priority, the system not working.

**Comment by estebangarcia** [comment]
Experiencing the same issue. Seeing this on chrome's dev console:

```
TypeError: Cannot read property 'toLowerCase' of undefined
    at patchManifestStage_PatchStatus.manifestIdentifier (providers~app.js:1)
    at manifests.map.m (providers~app.js:1)
    at Array.map (<anonymous>)
    at patchManifestStage_PatchStatus.componentDidUpdate (providers~app.js:1)
    at patchManifestStage_PatchStatus.componentDidMount (providers~app.js:1)
    at Hi (vendor_R_Z~app.js:1)
    at Fi (vendor_R_Z~app.js:1)
    at Gi (vendor_R_Z~app.js:1)
    at ii (vendor_R_Z~app.js:1)
    at xg (vendor_R_Z~app.js:1)
    at Object.enqueueSetState (vendor_R_Z~app.js:1)
    at Execution_Execution.G.setState (vendor_R_Z~app.js:1)
    at Execution_Execution.updateViewStateDetails (core~app.js:1)
    at SafeSubscriber.stateChangeSuccessSubscription.reactShims.i.stateEvents.stateChangeSuccess.subscribe [as _next] (core~app.js:1)
    at SafeSubscriber.__tryOrUnsub (vendor_R_Z~app.js:1)
    at SafeSubscriber.next (vendor_R_Z~app.js:1)
    at Subscriber._next (vendor_R_Z~app.js:1)
    at Subscriber.next (vendor_R_Z~app.js:1)
    at Subject.next (vendor_R_Z~app.js:1)
    at $rootScope.$on (core~app.js:1)
    at Scope.$broadcast (vendor_A_F~app.js:1)
    at $transition$.onSuccess.priority (vendor_P_Q~app.js:1)
    at TransitionHook.invokeHook (vendor_P_Q~app.js:1)
    at vendor_P_Q~app.js:1
    at Array.forEach (<anonymous>)
    at TransitionHook.runAllHooks (vendor_P_Q~app.js:1)
    at vendor_P_Q~app.js:1
    at vendor_A_F~app.js:1
    at vendor_A_F~app.js:1
    at Scope.$digest (vendor_A_F~app.js:1)
    at vendor_A_F~app.js:1
    at completeOutstandingRequest (vendor_A_F~app.js:1)
    at vendor_A_F~app.js:1 undefined
```

**Comment by maggieneterval** [comment]
Hi @rverma-nikiai and @estebangarcia, I really appreciate you reporting this issue. The cause is some unsafe lookups on a manifest's `apiGroup`, which is not present on the context of a Patch Manifest stage, added with [this change](https://github.com/spinnaker/deck/commit/57c30b5c762176acc85e215ddd614fd20b93b764). I've just put in a fix for this and we'll patch this into 1.13.

**Comment by estebangarcia** [comment]
Thanks for your prompt response and solution @maggieneterval
